doc/nrf/protocols/thread/tools.rst

```diff
@@ -87,7 +87,7 @@ To program the nRF device with the RCP application, complete the following steps
 
          .. code-block:: console
 
-            west build -p always -b nrf52840dongle_nrf52840 nrf/samples/openthread/coprocessor/
+            west build -p always -b nrf52840dongle_nrf52840 nrf/samples/openthread/coprocessor/ -S usb
```

Comment:
after https://github.com/nrfconnect/sdk-nrf/pull/11593 `usb` snippet should not be required for `nrf52840dongle_nrf52840` build target. Does the sample break without it?